pkgs/tools/security/ghidra/build.nix

```diff
@@ -99,18 +143,14 @@ HERE
   };
 
 in stdenv.mkDerivation {
```

Comment:
needs to be `stdenv.mkDerivation (finalAttrs: {` then add closing paren on the same line as the other `}`